<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>428</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Прокат автомобилей</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#tabWidget{
background-color: rgb(170, 170, 255);
}
QWidget#tabCar{
background-color: rgb(170, 170, 255);
}
QWidget#tabCustomer{
background-color: rgb(170, 170, 255);
}
QWidget#tabRent{
background-color: rgb(170, 170, 255);
}
QWidget#tabReport{
background-color: rgb(170, 170, 255);
}
QTabBar::tab:selected {
    background-color: rgb(170, 170, 255);
    
}</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabCar">
       <attribute name="title">
        <string>Автомобили</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,1">
        <item row="0" column="0">
         <widget class="QTableView" name="tableCar">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.510583, y1:1, x2:0.51, y2:0, stop:0 rgba(188, 255, 255, 255), stop:0.739583 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,8">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddCar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnAddCar{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnAddCar:hover{
background-color: rgb(61, 255, 255);
}
QPushButton#btnAddCar:focus{
background-color: rgb(61, 255, 255);
outline: none;
box-shadow: 0 0 0 4px #cbd6ee;
}
</string>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelCar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnDelCar{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnDelCar:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditCar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnEditCar{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnEditCar:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCustomer">
       <attribute name="title">
        <string>Клиенты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="4,1">
        <item row="0" column="0">
         <widget class="QTableView" name="tableCustomer">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.510583, y1:1, x2:0.51, y2:0, stop:0 rgba(188, 255, 255, 255), stop:0.739583 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,2,8">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddCustomer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnAddCustomer{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnAddCustomer:hover{
background-color: rgb(61, 255, 255);
}

</string>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelCustomer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnDelCustomer{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnDelCustomer:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditCustomer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnEditCustomer{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnEditCustomer:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRent">
       <attribute name="title">
        <string>Аренда</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="4,1">
        <item row="0" column="0">
         <widget class="QTableView" name="tableRent">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.510583, y1:1, x2:0.51, y2:0, stop:0 rgba(188, 255, 255, 255), stop:0.739583 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,2,8">
          <item>
           <widget class="QPushButton" name="btnAddRent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnAddRent{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnAddRent:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Аренда</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelRent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnDelRent{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnDelRent:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Возврат</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditRent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnEditRent{
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
background-color: rgb(185, 255, 255);
font-weight:bold;
}
QPushButton#btnEditRent:hover{
background-color: rgb(61, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReport">
       <attribute name="title">
        <string>Отчётность</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableReport">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.510583, y1:1, x2:0.51, y2:0, stop:0 rgba(188, 255, 255, 255), stop:0.739583 rgba(255, 255, 255, 255));</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
